### ISSUE TITLE
StaticInjectorError

### ISSUE THREAD

**Issue by Nikhilkapoor20** [fulls1z3/ngx-cache#87]
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[x] Support request
[ ] Feature request
[ ] Documentation issue or request

**Current behavior**
After following the steps, when I used cacheService in my project. I am getting following error

StaticInjectorError(AppModule)[InjectionToken CACHE]: 
  StaticInjectorError(Platform: core)[InjectionToken CACHE]: 
    NullInjectorError: No provider for InjectionToken CACHE!

* I am using this with the latest version of angular
* I have added my service to the provider.

* **Browser:**
- [x] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX

**Comment by Tho77** [comment]
Getting the same error trying to do the basic implementation.

**Comment by mato-stanic** [comment]
Had same issue, until I got through documentation one more time and read it carefully. 
Take a look at:
 https://github.com/fulls1z3/ngx-cache/tree/master/packages/%40ngx-cache/core#-setting-up-cachemodule-to-use-cachestaticloader 
there is a note below 
"Note: You need to perform browser (@ngx-cache/platform-browser), and (for Angular Universal) server (@ngx-cache/platform-server) platform implementations to begin using caching features."

Basically I have used [ngx-cache/platform-browser](https://github.com/fulls1z3/ngx-cache/tree/master/packages/@ngx-cache/platform-browser) with LocalStorageCacheService. Hopefully this will help you.

**Comment by k-vekos** [comment]
What do I do if I have disabled server-side prerendering in my application so I have no server, just client?

**Comment by mmealling** [comment]
@mato-stanic What does your app.module.ts look like? I'm following the documentation for an SPA using LocalStorageCacheService and I'm getting something similar. I've upgraded to Angular 6.
```
Error: StaticInjectorError(AppModule)[BrowserCacheModule -> TransferState]: 
  StaticInjectorError(Platform: core)[BrowserCacheModule -> TransferState]: 
    NullInjectorError: No provider for TransferState!
    at NullInjector.push../node_modules/@angular/core/fesm5/core.js.NullInjector.get (core.js:979)
    at resolveToken (core.js:1232)
    at tryResolveToken (core.js:1182)
    at StaticInjector.push../node_modules/@angular/core/fesm5/core.js.StaticInjector.get (core.js:1077)
    at resolveToken (core.js:1232)
    at tryResolveToken (core.js:1182)
    at StaticInjector.push../node_modules/@angular/core/fesm5/core.js.StaticInjector.get (core.js:1077)
    at resolveNgModuleDep (core.js:9238)
    at _createClass (core.js:9279)
    at _createProviderInstance$1 (core.js:9255)
```

**Comment by mmealling** [comment]
After poking around in the seed project I found that you need a bit more in your app.module.ts:

First, import BrowserTransferStateModule and TransferState from the @angular/platform-browser:
``
import { BrowserTransferStateModule, TransferState } from '@angular/platform-browser';
```
Then add BrowserTransferStateModule to imports.
